javascript/cw_tool.path.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * @file
+ *  Add path related behaviours.
+ */
+var CW = CW || {};
+
+(function($) {
```

Comment:
Why adding $? $ is an old standard and hides the owner. Nowadays $ is used by other libraries than jQuery.